# Recommended literature

Here we have gathered some literature on metabarcoding.  
- [Zinger et al. 2019](Zinger_et_al_2019.pdf). DNA metabarcoding—Need for robust experimental designs to draw sound ecological conclusions. Molecular Ecology, 28, 1857-1862
- [Deiner et al. 2017](Deiner_et_al-2017-Molecular_Ecology.pdf). Environmental DNA metabarcoding: Transforming how we survey animal and plant communities. Molecular Ecology, 26, 5872-5895.
- [Bohmann et al. 2014](Bohmann_et_al_2014) . Environmental DNA for wildlife biology and biodiversity monitoring. TREE, 29.
- [Alberdi et al. 2017](Alberdi_et_al_2017.pdf). Scrutinizing key steps for reliable metabarcoding of environmental samples. Methods in Ecology and Evolution, 9, 134-147.
- Ficetola et al. 2016. How to limit false positives in environmental DNA and metabarcoding? Molecular Ecology, 16, 604-607.
- [Dickie et al. 2018](Dickie_et_al_2018.pdf). Towards robust and repeatable sampling methods ineDNA-based studies. Molecular Ecology Resources, 18, 940-952. Schnell et al. 2015. Tag jumps illuminated – reducing sequence‐to‐sample misidentifications in metabarcoding studies. Molecular Ecology Resources, 15, 1289-1303.
- [Antich et al. 2021](Antich_et_al_2021.pdf). To denoise or to cluster, that is not the question: optimizing pipelines for COI metabarcoding and metaphylogeography. BMC Bioinformatics, 22,177.
- [Lamb et al. 2019](Lamb_et_al_2018.pdf). How quantitative is metabarcoding: A meta‐analytical approach. Molecular Ecology, 28, 420-430.
- [Jamy et al. 2019](Jamy_et_al_2019.pdf). Long-read metabarcoding of the eukaryotic rDNA operon to phylogenetically and taxonomically resolve environmental diversity. Molecular Ecology Resources, 20, 429-443.
-[Schnell et al. 2015](Schnell_et_al_2015). Tag jumps illuminated – reducing sequence‐to‐sample misidentifications in metabarcoding studies. Molecular Ecology Resources, 15, 1289-1303.Antich et al. 2021. To denoise or to cluster, that is not the question: optimizing pipelines for COI metabarcoding and metaphylogeography. BMC Bioinformatics, 22,177.


### Methods
- [Foster et al. 2017](Foster_et_al_2017.pdf). Metacoder: An R package for visualization and manipulation of community taxonomic diversity data. PLoS Computational biology, 13.
- [Rognes et al. 2016](Rognes_et_al_2016.pdf). VSEARCH: a versatile open source tool for metagenomics. PeerJ
- [Mahe et al. 2015](Mahe_et_al_2015.pdf). Swarm v2: highly-scalable and high-resolution amplicon clustering. PeerJ
- [Callahan et al. 2016](Callahan_et_al_2016.pdf). DADA2: High-resolution sample inference from Illumina amplicon data. Nature Methods, 13, 581-588.
- [Barbera et al. 2019](Barbera_et_al_2019.pdf). EPA-ng: Massively Parallel Evolutionary Placement of Genetic Sequences. Systematic Biology, 68, 365-369.
- [Deiner et al. 2016](Deiner_et_al_2016.pdf) - Environmental DNA reveals that rivers are conveyer belts of biodiversity information. Nat. Com


### Networks
- [Röttjers et al. 2018](Rottjers_2018.pdf) - From hairballs to hypotheses–biological insights from microbial networks. FEMS Microbiology Reviews, 42, 761–780
- [Layeghifard et al.2017](Layeghifard_2017.pdf) - Disentangling Interactions in the Microbiome: A Network Perspective. Trend. Microbiol.
- [Faust et al. 2012](Faust_nrmicro2832.pdf)  Microbial interactions: from networks to models. Nat. rev. Microb.

### Data Visualisation
- [R for Data Science, Wickham and Grolemund](https://r4ds.had.co.nz/) - Free ebook
- [R graphics Cookbook, W. Chang](https://r-graphics.org/) - Free ebook 

### Other links
- [Youtube: High Resolution Metagenomic Sequencing using DADA2 and LoopSeq](https://youtu.be/eV_UCr-5ARE) - B. Callahan